### PR TITLE
Add Azure centos base image build job

### PR DESCRIPTION
**Pull request by pschiffe**

### CODE REVIEW

**Comment by pschiffe** [comment]
@kargakis

File: sjb/config/test_cases/azure_build_base_image_centos.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+  - type: "host_script"
+    title: "move secrets to remote host"
+    script: |-
+      rsync --copy-links --omit-dir-times --archive --rsh "ssh -F ./.config/origin-ci-tool/inventory/.ssh_config" /var/lib/jenkins/.azure/* openshiftdevel:/home/origin/release/cluster/test-deploy/${CLUSTER_PROFILE}/
+      ssh -F ./.config/origin-ci-tool/inventory/.ssh_config openshiftdevel 'sudo chown -R origin:origin-git /home/origin/release/cluster/test-deploy/${CLUSTER_PROFILE}/'
+      ssh -F ./.config/origin-ci-tool/inventory/.ssh_config openshiftdevel 'sudo chmod -R ug+rwX /home/origin/release/cluster/test-deploy/${CLUSTER_PROFILE}/'
+
+  - type: "script"
+    title: "build node image"
```
Comment (kargakis) [inline sjb/config/test_cases/azure_build_base_image_centos.yml]: s/node/base/

**Comment by kargakis** [comment]
Do you have a PR for openshift/release?

**Comment by pschiffe** [comment]
@kargakis what do you mean? I've updated our azure branch to support also centos

**Comment by kargakis** [comment]
@pschiffe we need a PR to run this periodically in prow, similar to https://github.com/openshift/release/pull/778

**Comment by pschiffe** [comment]
Got it: https://github.com/openshift/release/pull/793

**Pull request merged**